app/assets/stylesheets/alchemy/sitemap.scss

```diff
@@ -1,3 +1,107 @@
+#sitemap {
+
+  &.list {
+    padding: 16px 12px;
+  }
+
+  &.sorting {
+
+    .page.icon { cursor: move }
+  }
+
+  .page.icon {
+    float: left;
+    position: relative;
+  }
+
+  .locked .page.icon {
+    background-position: -32px -40px;
+  }
+
+  .legend {
+    color: #666;
+    line-height: 2em;
+
+    .node_url { padding-right: 84px }
+    .node_name { margin-left: 32px }
+  }
+
+  .root_node {
+    font-weight: bold;
+    margin-bottom: 1em;
+
+    &:hover {
+      .sitemap_right_tools { display: block }
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `sitemap_right_tools` should be written in lowercase with hyphens<br>Declaration should be terminated by a semicolon